hooks/scan_scene_tk-mari.py

```diff
@@ -71,13 +71,16 @@ def execute(self, **kwargs):
 
                 params = {"geo":geo.name(), "channel":channel.name()}
 
+                # find all publishable layers:
+                publishable_layers = self.find_publishable_layers_r(channel.layerList())
+                if not publishable_layers:
+                    # no layers to publish!
```

Comment:
Worth adding a log_warning here ? From what I understand, this shouldn't happen, and we are not able to reproduce it, so having a bit of debug would help to resolve the problem properly ? No offense here, but the fix looks more like a workaround rather than a proper fix ?